front/src/components/Table/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+} from "@material-ui/core";
+
+import { selectBooksBasket } from "features/books/booksSlice";
+
+import { IconsList, SettingsIcon } from "./components";
+import { useStyles, InfoElement } from "./styles";
+
+interface IBook {
+  author: string;
+  cover_url: string;
+  currency: string;
+  id: number;
+  pages: number;
+  price: number;
+  title: string;
+  value: number;
+}
+
+export default function BasicTable() {
+  const basket = useSelector(selectBooksBasket);
+
+  const classes = useStyles();
+
+  const newRows = basket.map((book: IBook) => {
```

Comment:
brzydko to wygląda, lepiej
const newRows = basket.map((book: IBook) => ({
author: book.author
...
})

i dobrze wyniesc to do jakiegos helpera poza komponent

Reply:
Done.